test/puppetlabs/trapperkeeper/bootstrap_test.clj

```diff
@@ -154,21 +156,70 @@ puppetlabs.trapperkeeper.examples.bootstrapping.test-services/hello-world-servic
 puppetlabs.trapperkeeper.examples.bootstrapping.test-services/foo-test-service ; comment"
           service-maps      (->> bootstrap-config
                                  (StringReader.)
-                                 parse-bootstrap-config!
+                                 vector
+                                 parse-bootstrap-configs!
                                  (map service-map))]
       (is (= (count service-maps) 2))
       (is (contains? (first service-maps) :HelloWorldService))
       (is (contains? (second service-maps) :TestService)))))
 
+(deftest multiple-bootstrap-files
+  (testing "Multiple bootstrap files can be specified directly on the command line"
+    (with-test-logging
+      (let [bootstrap-one "./dev-resources/bootstrapping/cli/split_bootstraps/one/bootstrap_one.cfg"
+            bootstrap-two "./dev-resources/bootstrapping/cli/split_bootstraps/two/bootstrap_two.cfg"
+            bootstrap-path (format "%s,%s" bootstrap-one bootstrap-two)
+            app (bootstrap-with-empty-config ["--bootstrap-config" bootstrap-path])
+            test-svc (get-service app :TestService)
+            hello-world-svc (get-service app :HelloWorldService)]
+        (is (logged?
+              (format "Loading bootstrap configs:\n%s\n%s"
+                      (fs/absolute bootstrap-one)
+                      (fs/absolute bootstrap-two))
+              :debug))
+        (is (= (test-fn test-svc) :cli))
+        (is (= (hello-world hello-world-svc) "hello world")))))
+  (testing "A path containing multiple .cfg files can be specified on the command line"
+    (with-test-logging
+      (let [bootstrap-path "./dev-resources/bootstrapping/cli/split_bootstraps/both/"
+            app (bootstrap-with-empty-config ["--bootstrap-config" bootstrap-path])
+            test-svc (get-service app :TestService)
+            hello-world-svc (get-service app :HelloWorldService)]
+        (is (logged?
+              ; We can't know what order it will find the files on disk, so just
+              ; look for a partial match with the path we gave TK.
+              (re-pattern (format "Loading bootstrap configs:\n%s"
+                                  (fs/absolute bootstrap-path)))
+              :debug))
+        (is (= (test-fn test-svc) :cli))
+        (is (= (hello-world hello-world-svc) "hello world")))))
+  (testing "A path containing both a file and a folder can be specified on the command line"
```

Comment:
Yeah they're definitely being allowed (maybe by accident) right now, and just talked to Chris and he agrees that duplicate entries should be allowed. I'll add a test for duplicates